### PR TITLE
Add instructor informational page.

### PR DESCRIPTION
**Pull request by drapergeek**
All feedback is appreciated! I wanted something that we can email out a
link to the instructors rather than having an email that needs to be
repeated for each workshop.

### CODE REVIEW

File: views/workshop_info/instructor_information.erb [hunk 1]
```diff
@@ -0,0 +1,48 @@
+<div class="row">
+  <div class="span12">
+    <h2>Thank you!</h2>
+    <p>Seriously, we can't say it enough, without the help of our TAs, this workshop would not exist. We appreciate each and every one of you.</p>
```
Comment (mxie) [inline views/workshop_info/instructor_information.erb]: `Seriously, we can't say it enough - without the help of our TAs,`

File: views/workshop_info/instructor_information.erb [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+  <div class="span12">
+    <h2>Arrival Times</h2>
+    <p>We ask that TAs arrive 30 minutes prior to the workshop on Friday night and Saturday morning. This gives us plenty of time to introduce ourselves and get situated before the attendeees arrive.</p>
+  </div>
+</div>
+
+<div class="row">
+  <div class="span12">
+    <h2>What to expect</h2>
+    <p>An awesome time! RailsBridge is a great chance to reach out to a new group of people and show them how amazing our community is. You will meet and speak with people from all walks of life and backgrounds. The main thing is listen and help in anyway you can. You will not always know the answer and that is OK! Feel free to say you don't know and ask another TA or even google the answer!</p>
```
Comment (mxie) [inline views/workshop_info/instructor_information.erb]: Capitalize `Google`.

File: views/workshop_info/instructor_information.erb [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+    <p>It is highly recommended that if you have not been to a RailsBridge workshop before that you read over the curriculm ahead of time so that you have a general understanding of what the students will be doing. Your primary responsibility as a TA is to help the students along as they get stuck so knowing where they were heading is really helpful.</p>
```
Comment (mxie) [inline views/workshop_info/instructor_information.erb]: `...been to a RailsBridge workshop before, then you should read over the curriculum...`

The changes:
- s/before that you/before, then you should/
- correct misspelling of curriculum

File: views/workshop_info/instructor_information.erb [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+  </div>
+</div>
+
+<div class="row">
+  <div class="span12">
+    <h2>Extending the Learning</h2>
+    <p>One thing that comes up is that some students get very bored with just going through the tutorial. The best way for us to alleviate this is to extend their learning with some in person conversations. As people finish up their sections, take the opportunity to help extend their learning and teach them some extra things about what they just learned. Do not try and pressure them but if they seem to be enjoying what their doing and want to learn more, add in your own experiences into learning!</p>
```
Comment (mxie) [inline views/workshop_info/instructor_information.erb]: s/in person/in-person/

`what their doing` -> `what they're doing`

File: views/workshop_info/instructor_information.erb [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+      <li>"We just used a has_many and belongs_to association. Let's talk about some other things that may use that same type of relation."</li>
+      <li>"We allowed people to vote, but now anyone can vote twice. How might we prevent that from happening?"</li>
+    </ul>
+  </div>
+</div>
+
+<div class="row">
+  <div class="span12">
+    <h2>Keep Helping!</h2>
+    <p>Undoubtedly while you are going through the curriculm this weekend you find errors or places that could be improved. Please let us know! Feel free to put in a GitHub issue, Pull Request or just send us an email. The best way for us to help our students is to continually improve the curriculum and our TAs are the front line for that knowledge.</p>
```
Comment (mxie) [inline views/workshop_info/instructor_information.erb]: `curriculm` -> `curriculum`

**Comment by slothbear** [comment]
Is there a convention regarding long lines? It would be easier to track changes if the text had a smaller line length. 80? 72?

**Comment by slothbear** [comment]
and I meant to say... good idea having this info in a standard place.

**Comment by bcarreno** [comment]
This is a great idea.  And the content is better than what we were using.

**Pull request merged**